src/renderer-backend-egl.c

```diff
@@ -138,6 +138,12 @@ wpe_renderer_backend_egl_target_frame_rendered(struct wpe_renderer_backend_egl_t
     target->base.interface->frame_rendered(target->base.interface_data);
 }
 
+void
+wpe_renderer_backend_egl_target_deinitialize(struct wpe_renderer_backend_egl_target* target)
+{
+    target->base.interface->deinitialize(target->base.interface_data);
```

Comment:
Shouldn't we check the pointer before using it, to avoid breaking existing implementations which may not provide an implementation? Something like this, to avoid breaking the API expectations:

```suggestion
    if (target->base.interface->deinitialize)
        target->base.interface->deinitialize(target->base.interface_data);
```